Logger/BaseLogger.cs

```diff
@@ -2,6 +2,8 @@
 
 public abstract class BaseLogger
 {
-    public abstract void Log(LogLevel logLevel, string message);
+    public abstract void Log(LogLevel LogLevel, string Message);
```

Comment:
Don't we use camelCase for variable names? Maybe I'm mistaken here.
```suggestion
    public abstract void Log(LogLevel logLevel, string message);
```

Reply:
Yes a little mistake I found to late on Monday to fix had it in my plans thanks for the catch